env/production/oidc/terragrunt.hcl

```diff
@@ -0,0 +1,7 @@
+terraform {
+  source = "../../../aws//oidc"
```

Comment:
Could you switch this up to use the `TARGET_VERSION` env var:
```hcl
terraform {
  source = "git::https://github.com/cds-snc/forms-terraform//aws/oidc?ref=${get_env("TARGET_VERSION")}"
}
```

Reply:
> Did you also want to add this to the Production `terraform plan/apply` workflows?
> 
> ```
> .github/workflows/terragrunt-plan-production.yml
> .github/workflows/terragrunt-apply-production.yml
> ```

I was thinking that we'd probably want to verify that it works in staging first before modifying production, i'm not 100% clear on what rights are needed for running the plan step. 

Although I guess just because we create the roles doesn't mean we have to use them.